cg/services/bcl_convert_metrics_service/parser.py

```diff
@@ -130,6 +144,22 @@ def calculate_mean_quality_score(cls, metrics: list[SequencingQualityMetrics]) -
             total_q_score += metric.mean_quality_score_q30
         return round(total_q_score / SCALE_TO_READ_PAIRS, 2)
 
+    @classmethod
+    def calculate_mean_yield(cls, metrics: list[SequencingQualityMetrics]) -> float:
+        """Calculate the mean yield for a list of metrics."""
+        total_yield: float = 0
+        for metric in metrics:
+            total_yield += metric.yield_
+        return round(total_yield / SCALE_TO_READ_PAIRS, 2)
```

Comment:
Are you sure about the scalers? I would intuitively think we don't need to scale at all?

Reply:
one sample from the metrics file:

``` 
Lane,SampleID,Sample_Project,index,index2,ReadNumber,Yield,YieldQ30,QualityScoreSum,Mean Quality Score (PF),% Q30
1,REDACTEDSample1,405887,GTCCTTCGGC,CTGTGCATGA,1,415032696,393745856,15004333259,36.15,0.95
1,REDACTEDSample1,405887,GTCCTTCGGC,CTGTGCATGA,2,1436651640,1344252463,51555254683,35.89,0.94
```

Reply:
It is the same for all metrics, and it is the behaviour we do for all caluclations

Reply:
Or do we want to change this behaviour in regards to yield?

Reply:
We need to take the sum here